bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/StreamingBigtableResultScanner.java

```diff
@@ -137,16 +69,12 @@ public void complete() {
   @Override
   public Row next() throws IOException {
     Optional<Row> next = responseQueueReader.getNextMergedRow();
-    if (next.isPresent()) {
-      return next.get();
-    } else {
-      return null;
-    }
+    return next.isPresent() ? next.get() : null;
```

Comment:
If getNextMergedRow() return null or a full row, then we don't need to translate from !isPresent to null.